src/melee/mp/mpcoll.c

```diff
@@ -1135,7 +1145,7 @@ asm void func_80043C6C(CollData* arg0, s32 arg1, s32 arg2)
 } // clang-format on
 #pragma pop
 
-#else
+#elif false
```

Comment:
huh

Reply:
Bad signature in the function. Feel free to match it instead.

Reply:
Ah okay we can leave it for now